### PR TITLE
Reorganise information in lazer-related articles

### PR DESCRIPTION
**Pull request by Walavouchey**
closes https://github.com/ppy/osu-wiki/issues/9300

suggestions welcome

### CODE REVIEW

**Review by TicClick — CHANGES_REQUESTED** [review]

File: wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md [hunk 1]
```diff
@@ -1,61 +1,12 @@
...
-- Snaking out sliders in the osu! mode
-- No black bar behind the playfield in osu!taiko
-- Timing-based note colouring in osu!mania
-
-## Gameplay differences 
-
-### General
-
-#### Song playback ramps up and down when pausing and unpausing
+# Gameplay differences in osu!(lazer)
```
[the remark this answers is not in the document]
Comment (Walavouchey) [reply wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md:1]: as an article name that sounds not very descriptive

File: wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md
[the remark this answers is not in the document]
Comment (Walavouchey) [reply wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md:1]: they look like this on the website and it's intentional

image

File: wiki/Help_centre/Upgrading_to_lazer/en.md [hunk 2]
```diff
@@ -287,7 +313,7 @@ You're likely thinking of another game.
 [^countdown-timers-stable]: Set a countdown using a command, no automatic start.
 [^countdown-timers-lazer]: Set a countdown from the game UI to automatically start the match.
 [^queue-modes]: Turn on to allow anyone in a lobby to queue new beatmaps, a.k.a. "host rotate".
-[^difficulty-adjust]: Change CS/AR/OD/HP of a beatmap directly from song select.
+[^difficulty-adjust]: Change CS/AR/OD/HP of a beatmap directly from song select via the Difficulty Adjust mod.
```
[the remark this answers is not in the document]
Comment (Walavouchey) [reply wiki/Help_centre/Upgrading_to_lazer/en.md:316]: nope
image

File: wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md [hunk 3]
```diff
@@ -125,9 +76,30 @@ When a hit lands right at the edge of the hit window, a different judgement may
...
 | Needs further consideration | Yes |
 
-### osu!
+### Differences in scoring
+
+Scoring in lazer will use a system similar to ScoreV2, with all ScoreV1 scores converted into the new system.
+
+There are two interchangeable display modes for scores: *standardised* and *classic*. Standardised scoring limits score to a maximum of 1,000,000 points + bonus and score multipliers (similarly to ScoreV2), while classic scoring is the same as standardised, but scaled quadratically with the amount of hit objects in a beatmap (similar to ScoreV1). These can be selected from the settings, with all places were score is displayed in-game changing accordingly.
+
+There are also some differences in how much score each hit object and each judgement is responsible for relative to each other.
```
[the remark this answers is not in the document]
Comment (Walavouchey) [reply wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md:85]:
```suggestion
There are also some differences in how much hit objects and judgements contribute to a score relative to each other.
```

File: wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md
[the remark this answers is not in the document]
Comment (CloneWith) [reply wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md:1]: yep I understood that :heart

File: wiki/Help_centre/Upgrading_to_lazer/en.md [hunk 4]
```diff
@@ -39,22 +37,43 @@ The following is a comprehensive list of the **current state** of lazer in compa
...
 | In-game skin / UI layout editing | ![No][false] | ![Yes][true] |
+| Dynamic customisable skinning components | ![No][false] | ![Yes][true] |
+| Combo bursts and associated sounds | ![Yes][true] | ![No][false] |
+| Skinnable gameplay countdown | ![Yes][true] | ![No][false] |
+| Skinnable gameplay leaderboard | ![Yes][true] | ![No][false] |
+| Skinnable cutscene skip button | ![Yes][true] | ![No][false] |
+| Skinnable section fail/pass | ![Yes][true] | ![No][false] |
+| Skinnable results screen | ![Yes][true] | ![No][false] |
+| Skinnable song select interface components | ![Yes][true] | ![No][false] |
+| Skinnable pause menu | ![Yes][true] | ![No][false] |
```
Comment (peppy) [inline wiki/Help_centre/Upgrading_to_lazer/en.md:48]: I mean this is kinda covered by the "gameplay only" clause above. Not sure listing here does much other than spam the list.

**Review by peppy — COMMENTED** [review]
Looks mostly fine

File: wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (peppy) [reply wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md:85]: I'm not actually sure what this refers to specifically. Maybe worth elaborating on if it's to remain?

**Review by TicClick — APPROVED** [review]

File: wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Walavouchey) [reply wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md:85]: this is something on my list to investigate shortly (will report about it against `ppy/osu`, as i mentioned in the meeting), so i'll elaborate in another pass

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**